docs/appendices/release-notes/unreleased.rst

```diff
@@ -74,6 +74,9 @@ Changes
   function for improved PostgreSQL compatibility. CrateDB does not support
   comments for columns, so this function always returns ``NULL``.
 
+- Improved performance of ``COPY FROM`` or INSERTS with multi-values into
+  partitioned tables.
+
```

Comment:
Isn't this a bit misleading?  It improves the performance of statements that create multiple partitions at once.

Reply:
Yep, it's only bulk creation. 

I wanted to mention some user facing commands, I think I can just take entry from 
https://github.com/crate/crate/commit/9e12f8142debd84b7bc299ddeda84ab19332d9d7#diff-59130575b4fb2932c957db2922977d7d89afb0b2085357db1a14615a2fcad776R10
> Further improved bulk partition creation which can occur during ``COPY FROM`` or INSERTS with multi-values into partitioned tables
    
Updated the entry.